code_schemes/s01e01.json

```diff
@@ -0,0 +1,161 @@
+{
+    "SchemeID": "Scheme-03bb299c",
+    "Name": "S01E01",
+    "Version": "0.0.0.1",
+    "Codes": [
+      {
+        "CodeID": "code-E-720cdb66",
+        "CodeType": "Meta",
+        "MetaCode": "escalate",
+        "DisplayText": "meta: ESCALATE",
+        "NumericValue": -100080,
+        "StringValue": "escalate",
+        "VisibleInCoda": true,
+        "Shortcut": "e"
+      },
```

Comment:
Check with @lukechurch about whether to anticipate any other codes here.

Reply:
Thanks will do.

Reply:
Merging to test on Miranda